templates/experiments/single_experiment_home.html

```diff
@@ -311,6 +312,8 @@ <h3 class="font-bold text-lg">Parent Experiments:
       </div>
     </div>
   </div>
+  {% endif %}
```

Comment:
I think this should be one line above to be consistent with the tab `<div>` elements